HtmlMesh/example.js

```diff
@@ -0,0 +1,226 @@
+                <option value="Australia">Australia</option>
+            </select><br><br>
+            
+            <label for="hobbies">Hobbies:</label><br>
+            <input type="checkbox" id="hobby1" name="hobbies" value="Reading">
+            <label for="hobby1">Reading</label><br>
+            <input type="checkbox" id="hobby2" name="hobbies" value="Gaming">
+            <label for="hobby2">Gaming</label><br>
+            <input type="checkbox" id="hobby3" name="hobbies" value="Sports">
+            <label for="hobby3">Sports</label><br><br>
+        </form>
+    `;
+    div.style.backgroundColor = 'white';
+    div.style.width = '480px';
+    div.style.height = '360px';
+    // Style the form
+    
+    htmlMeshDiv.setContent(div, 4, 3);
+    htmlMeshDiv.position.x = -3;
+    htmlMeshDiv.position.y = 2;
+
+    // Shows how this can be used to include a PDF in your scene.  Note this is 
+    // conceptual only.  Displaying a PDF like this works, but any links in the
+    // PDF will navigate the current tab, which is probably not what you want.
+    // There are other solutions out there such as PDF.js that may give you more
+    // control, but ultimately proper display of PDFs is not within the scope of
+    // this project.
+    const pdfUrl = 'https://cdn.glitch.com/3da1885b-3463-4252-8ded-723332b5de34%2FNew_Horizons.pdf#zoom=200?v=1599831745689'
+    const htmlMeshPdf = new HtmlMesh(scene, "html-mesh-pdf");
+    const iframePdf = document.createElement('iframe');
+    iframePdf.src = pdfUrl;
+    iframePdf.width = '480px';
+    iframePdf.height = '360px';
+    htmlMeshPdf.setContent(iframePdf, 4, 3);
+    htmlMeshPdf.position.x = 3;
+    htmlMeshPdf.position.y = 2;
+
+    // Shows how this can be used to include a website in your scene
+    const siteUrl = 'https://www.babylonjs.com/';
+    const htmlMeshSite = new HtmlMesh(scene, "html-mesh-site");
+    const iframeSite = document.createElement('iframe');
+    iframeSite.src = siteUrl;
+    iframeSite.width = '480px';
+    iframeSite.height = '360px';
+    htmlMeshSite.setContent(iframeSite, 4, 3);
+    htmlMeshSite.position.x = -3;
+    htmlMeshSite.position.y = -2;
+    
+    // Shows how this can be used to include a YouTube video in your scene
+    const videoId = 'zELYw2qEUjI';
+    const videoUrl = [ 'https://www.youtube.com/embed/', videoId, '?rel=0&enablejsapi=1&disablekb=1&controls=0&fs=0&modestbranding=1' ].join( '' );
+    const htmlMeshVideo = new HtmlMesh(scene, "html-mesh-video");
+    const iframeVideo = document.createElement('iframe');
+    iframeVideo.src = videoUrl;
+    iframeVideo.width = '480px';
+    iframeVideo.height = '360px';
+    htmlMeshVideo.setContent(iframeVideo, 4, 3);
+    htmlMeshVideo.position.x = 3;
+    htmlMeshVideo.position.y = -2;
+
+    // Shows an example of how you can mask pointer events for a mesh, in this example the disc
+    // If you allow rotation, you'd have to use CSS transforms to align the mask with the disc
```

Comment:
Should this part be a separated example ?

Reply:
Yes, I can remove.  I started to create an example of how to do pointer masking but then realized you'd have to CSS transform the mask and so I left it for something to do maybe do later.